src/elo/util/dbInterface.ts

```diff
@@ -78,6 +88,9 @@ export const pullMatches = async (
           results: true,
         }
       }
+    },
+    orderBy: {
+      timestamp: "asc",
```

Comment:
Thank you for catching this! Vitally important element that I missed in my first pass.